packages/sdk/src/react/hooks/state.ts

```
@@ -0,0 +1,73 @@
+}
+
+/**
+ * Custom hook to retrieve all entities that have a specific model.
+ *
+ * @param model - The model to retrieve, specified as a string in the format "namespace-modelName".
+ * @returns The model structure if found, otherwise undefined.
+ */
+export function useModels<
+    N extends keyof SchemaType,
+    M extends keyof SchemaType[N] & string,
+    Client extends (...args: any) => any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)